plugin/src/main/java/com/denizenscript/denizen/events/world/SpawnChangeScriptEvent.java

```diff
@@ -36,23 +36,20 @@ public SpawnChangeScriptEvent() {
 
     @Override
     public boolean matches(ScriptPath path) {
-        if (!path.tryObjectSwitch("world", new WorldTag(event.getWorld()))) {
+        if (!path.tryObjectSwitch("for", new WorldTag(event.getWorld()))) {

```

Comment:
Backwards: This has been `world` for years, it's the meta and the register that appear to have been updated wrongly